frb_codegen/src/generator_dart.rs

```diff
@@ -361,7 +361,7 @@ fn generate_wire2api_func(ty: &ApiType, api_file: &ApiFile) -> String {
             }
         },
         Optional(opt) => format!(
-            "return raw == null ? null : _wire2api_{}(raw);",
+            "if (raw != null) return _wire2api_{}(raw);",
```

Comment:
What about keeping the original code: it is explicit about what it returns and is thus more readable especially for new Dart users. Btw if you want to keep this change, maybe put it in a new PR, which will make the diff of PRs more readable.

Reply:
Good point. I think I will revert this.